common/Makefile.am

```diff
@@ -58,6 +58,7 @@ libcommon_la_SOURCES = \
   log.h \
   os_calls.c \
   os_calls.h \
+  parse.c \
```

Comment:
The parse.c file is missing from this commit.

Reply:
Oops. Should be there now!